src/libs/TransactionUtils.ts

```diff
@@ -239,6 +239,11 @@ function getUpdatedTransaction(transaction: Transaction, transactionChanges: Tra
         shouldStopSmartscan = true;
     }
 
+    if (Object.hasOwn(transactionChanges, 'customUnitRateID')) {
+        updatedTransaction.modifiedCustomUnitRateID = transactionChanges.customUnitRateID;
+        shouldStopSmartscan = true;
```

Comment:
I don't have the context for this, and hence asking, this is needed to be true right?